api/models/Campaign.js

```diff
@@ -30,6 +31,12 @@ const schema = new mongoose.Schema({
   msg_menu_signedup: String,
   msg_no_photo_sent: String,
 
+  // Mobile Commons Specific Fields.
+  mobileCommonsGroups: {
```

Comment:
Same here -- all other models are using lowercase/underscore. If we're creating a `mobilecommons` object, I do consider whether `keywords` should be moved into it too.